website/docs/BasicInterfaces/Agent.md

```diff
@@ -1,58 +1,45 @@
-# SecretAgent
+# Agent
 
-This is the primary entry point for launching SecretAgent. The following is a simple example:
+This is the primary class to interact with SecretAgent. The following is a simple example:
 
 ```js
-const SecretAgent = require('secret-agent');
+const agent = require('secret-agent');
 
 (async () => {
-  const agent = new SecretAgent();
   await agent.goto('https://www.google.com');
   // other actions...
   await agent.close();
 })();
 ```
 
```

Comment:
Jump into the "Agent class" through me for a loop when instead of a class example I was shown an instance... I'm not sure what to do here, but I find it difficult to make the jump.

Reply:
I wonder if we need to export the Agent class so it's clear that lowercase `agent` is an instance of Agent and not just a singleton that we lowercased.

Reply:
The example you've highlighted has the agent being exported from the secret-agent module right above. Did you miss that part?

I agree it's a bit jarring, but I think you come into the website if you start here, but I think you'll start on the Overview page, not here.

Reply:
I think you mean imported. Yes, I see that it's imported from the `secret-agent` library. That doesn't change anything for me.

I know we both agreed only the `agent` instance and Handler class should be exported from `secret-agent` (not Agent), but seeing the documentation is making me question that decision.

We've hidden some of the dots connecting how everything ties together. Yes, it's jarring, but it also just doesn't quite make sense.

I've been going back and forth on what to do about this. I wonder if we should just add a new "Default Export" page as the first item in Basic Interfaces.